Detectors/MUON/MCH/Simulation/include/MCHSimulation/Digitizer.h

```diff
@@ -76,6 +76,8 @@ class Digitizer
   o2::dataformats::MCTruthContainer<o2::MCCompLabel> mMCTruthOutputContainer;
 
   int processHit(const Hit& hit, int detID, double event_time);
+
+  int getGlobalDigit(int detID, int padID);
```

Comment:
I would not make this part of the interface at all. 
Whether or not we need a global digit id, the Digitizer is not the right place to define it anyway.
As the method access to private member of the class, it can be a free function defined only the cxx

Reply:
Hi,
It is now a free function.
let me know, if this is fine like this.